src/js/components/homepage/hero/Hero.jsx

```diff
@@ -9,10 +9,21 @@ import { InfoCircle } from 'components/sharedComponents/icons/Icons';
 import HeroButton from './HeroButton';
 import HeroTooltip from './HeroTooltip';
 
+// Fiscal year, document URL, and spending amount constants, updated yearly
+// last updated: 11/5/2018
+const FISCAL_YEAR = 2018;
+const FISCAL_DOCUMENT_URL = `https://www.fiscal.treasury.gov/fsreports/rpt/mthTreasStmt/mts0918.pdf`;
```

Comment:
According to the A/C, the link can stay the same year-to-year, so we can leave it hard-coded.